services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -5,6 +5,7 @@ type WorkerManagerWorkerPoolSummary {
     providerId: String!
     config: JSON
     emailOnError: Boolean!
+    currentCapacity: Int!
```

Comment:
Navigating to http://localhost:5080/worker-manager shows a red banner with text `Cannot return null for non-nullable field WorkerManagerWorkerPoolSummary.currentCapacity.`

Is this value not always defined?

Reply:
This will only work once the services with `currentCapacity` are deployed. It works when I run it against my dev deployment

Reply:
Makes sense. Thank you!